Sources/Datadog/RUM/RUMFeature.swift

```diff
@@ -105,7 +105,7 @@ internal final class RUMFeature {
                         device: commonDependencies.mobileDevice
                     ),
                     .ddAPIKeyHeader(clientToken: configuration.clientToken),
-                    .ddEVPOriginHeader(source: configuration.common.source),
+                    .ddEVPOriginHeader(source: configuration.common.origin ?? configuration.common.source),
```

Comment:
The reason for that is that we don't want to charge users by the usage of APM (and potentially also logs on the fututre) because of a test run, the way those charges are removed are by using the `origin` tag which must be transmitted in the header in case of end to end tests. This header is also used by synthetics for the same goal I think.